docs/apps/blender.md

```diff
@@ -0,0 +1,63 @@
+---
+tags:
+  - Free
+---
+
+# Blender
+
+Blender is the free, open source 3D creation suite which supports a variety of graphical applications such as 3D pipeline—modeling, rigging, animation, simulation, rendering, compositing and motion tracking, even video editing and game creation. 
+Advanced users can also use Blender’s API for Python scripting to customize the application and write specialized tools. 
+
+## Available
+
+* **Puhti**: 3.2, 3.6
+
+## License
+
+* [GPL](https://download.blender.org/release/GPL3-license.txt)
+
+## Usage
+
+You need remote graphics to work with Blender. 
+
+Using a web browser, go to [www.puhti.csc.fi](https://www.puhti.csc.fi). 
+
+On the landing page, click on "Log in" and select an appropriate authentication provider. When logging in using your CSC user account, select CSC as the authentication provider and use the same username and password you use when connecting with `ssh`.
+&nbsp;
+
+![Puhti web interface login page](../img/ood_login.png)
+
+After successful authentication, you will see the dashboard.
+&nbsp;
+
+![Puhti web interface front page](../img/ood_main.png)
+
+Select 'Accelerated Visualization' option from the dashboard.
+&nbsp;
+
+Set the required documentation and the resources needed for your job. Click "Launch" button to start your interactive session.
+&nbsp;
+
+![Puhti acclereated visualization page](../img/puhti_accelerated_visualization.png)
+
+Your session will be queued until the requested resources are available. This is indicated by the label 'Queued'. Once the resources are available, your session will launch. This is indicated by the label 'Running' 
+&nbsp;
+
+![Puhti web interface interactive session](../img/ood_puhti_interactive_session_launch.png)
```

Comment:
I think this part could be simplified/removed. These steps are already explained at https://docs.csc.fi/computing/webinterface/ and https://docs.csc.fi/computing/webinterface/accelerated-visualization/ so maybe just a link to those would suffice?

Reply:
I've now re-written the text and added links to existing pages.

Reply:
Thanks! Can you still please format the links at internal links? The tests are now failing because they are not.